roomdb/sqlite/invites.go

```diff
@@ -250,8 +250,12 @@ func (i Invites) List(ctx context.Context) ([]roomdb.Invite, error) {
 	return invs, nil
 }
 
-func (i Invites) Count(ctx context.Context) (uint, error) {
-	count, err := models.Invites().Count(ctx, i.db)
+func (i Invites) Count(ctx context.Context, onlyActive bool) (uint, error) {
+	queryMod := qm.Where("1")
```

Comment:
You can also do this, then we don't need this default and if clause:

```go
qry := qm.Where("active = ?", onlyActive)
```

Reply:
Thanks a lot for your feedback! I read through the SQLBoiler docs to see if I could avoid that. If I understand correctly this suggestion would return the number of inactive invites for `onlyActive=false`, not the original behaviour of inactive+active?

Reply:
ah.. you are totally right. My suggestion was half-assed, indeed.

What _should_ work is this:

```go
	// have an empty query mods slice
	var queryMod []qm.QueryMod
	if onlyActive { // append to it if we want to filter
		queryMod = append(queryMod, qm.Where("active = true"))
	}
	// expand the slice to the variadic arg (works fine with zero qm's)
	count, err := models.Invites(queryMod...).Count(ctx, i.db)
	...
```

Reply:
but i'm honestly not sure I prefer that, feels clunky..

 having the tautological `true == true` (ie `1`) in there doesn't seem bad either

 merging this. thanks again!

Reply:
Thanks you, I appreciate seeing your suggestions a lot!